plugin/CactbotEventSource/FFXIVProcessCn.cs

```diff
@@ -3,677 +3,759 @@
 using System.Runtime.InteropServices;
 using Newtonsoft.Json.Linq;
 
-namespace Cactbot {
-  public class FFXIVProcessCn : FFXIVProcess {
-    //
-    // for FFXIV CN version: 5.1
-    //
-    // Latest CN version can be found at:
-    // http://ff.sdo.com/web8/index.html#/patchnote
-    //
-    [StructLayout(LayoutKind.Explicit)]
-    public unsafe struct EntityMemory {
-      public static int Size => Marshal.SizeOf(typeof(EntityMemory));
+namespace Cactbot
+{
```

Comment:
Hmm, why are you reformatting this at the same time? If this is the same as FFXIVProcessIntl, then it should be the identical file other than FFXIVProcessCn vs FFXIVProcessIntl, if you don't mind.

Reply:
Visual Studio automatically formatted it for me and I didn't notice that ><